crm_location/README.rst

```diff
@@ -0,0 +1,46 @@
+CRM location
```

Comment:
AGPL badge is missing

Here you have a template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst